datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -326,19 +339,19 @@ def assert_metrics_using_metadata(self, metadata_metrics, check_metric_type=True
             if metric_name in exclude:
                 continue
             for metric_stub in metric_stubs:
-
-                if metric_stub.name not in metadata_metrics:
-                    errors.add("Expect `{}` to be in metadata.csv.".format(metric_stub.name))
+                normalized_metric_stub_name = backend_normalize_metric_name(metric_stub.name)
```

Comment:
`normalized_metric_stub_name` => `metric_stub_name` is maybe more readable and enough.